examples/crm/src/dataGenerator/contactNotes.ts

```diff
@@ -16,11 +14,10 @@ export const generateContactNotes = (db: Db): ContactNote[] => {
         return {
             id,
             contact_id: contact.id,
-            type: random.arrayElement(type),
```

Comment:
why no more type?

Reply:
None of the note components display or use this property. Or maybe I've missed it